lib/fetchers/tar.rb

```diff
@@ -28,6 +32,14 @@ def initialize(target)
       end
     end
 
+    def url
```

Comment:
it feels strange to have url and archive_path and target, since all refer to the same content.